modules/ossm-architecture.adoc

```diff
@@ -7,15 +7,12 @@
 
 {ProductName} is logically split into a data plane and a control plane:
 
-The *data plane* is a set of intelligent proxies deployed as sidecars. These proxies intercept and control all inbound and outbound network communication between microservices in the service mesh. Sidecar proxies also communicate with Mixer, the general-purpose policy and telemetry hub.
+The *data plane* is a set of intelligent proxies deployed as sidecars. These proxies intercept and control all inbound and outbound network communication between microservices in the service mesh. 
 
-* *Envoy proxy* intercepts all inbound and outbound traffic for all services in the service mesh. Envoy is deployed as a sidecar to the relevant service in the same pod.
+The architectural units used by previous versions have been replaced by Istiod. In 2.0 the control plane components Mixer, Pilot, Citadel, Galley and the sidecar injector functionality have been combined into a single component, Istiod.
 
-The *control plane* manages and configures proxies to route traffic, and configures Mixers to enforce policies and collect telemetry.
+Although Mixer is no longer supported as a control plane component, Mixer policy and telemetry plugins are now supported through WebAssembly, or WASM, extensions in Istiod. For more information about how to use WebAssembly extensions, see *Extensions*. Mixer can be enabled for policy and telemetry if you need to integrate legacy Mixer plugins.
```

Comment:
Since Mixer is deprecated, I don't think we should probably only mention it in the upgrade topic and not mention it as being part of the current architecture.  We don't really want new customers starting to use Mixer.